pvlib/tracking.py

```diff
@@ -570,3 +522,151 @@ def singleaxis(apparent_zenith, apparent_azimuth,
+    axis_azimuth : float
+        direction of tracker axes on horizontal [degrees]
+
+    Returns
+    -------
+    axis_tilt : float
+        tilt of tracker [degrees]
+
+    See also
+    --------
+    pvlib.tracking.singleaxis
+    pvlib.tracking.calc_cross_axis_tilt
+
+    Notes
+    -----
+    See [1]_ for derivation of equations.
+
+    References
+    ----------
+    .. [1] Kevin Anderson and Mark Mikofski, "Slope-Aware Backtracking for
+       Single-Axis Trackers", Technical Report NREL/TP-5K00-76626, July 2020.
+       https://www.nrel.gov/docs/fy20osti/76626.pdf
+    """
+    delta_gamma = axis_azimuth - slope_azimuth
+    # equations 18-19
+    tan_axis_tilt = cosd(delta_gamma) * tand(slope_tilt)
+    return np.degrees(np.arctan(tan_axis_tilt))
+
+
+def _calc_tracker_norm(ba, bg, dg):
+    """
+    Calculate tracker normal, v, cross product of tracker axis and unit normal,
+    N, to the system slope plane.
+
+    Parameters
+    ----------
+    ba : float
+        axis tilt [degrees]
+    bg : float
+        ground tilt [degrees]
+    dg : float
+        delta gamma, difference between axis and ground azimuths [degrees]
+
+    Returns
+    -------
+    vector : tuple
+        vx, vy, vz
+    """
+    cos_ba = cosd(ba)
+    cos_bg = cosd(bg)
+    sin_bg = sind(bg)
+    sin_dg = sind(dg)
+    vx = sin_dg * cos_ba * cos_bg
+    vy = sind(ba)*sin_bg + cosd(dg)*cos_ba*cos_bg
+    vz = -sin_dg*sin_bg*cos_ba
+    return vx, vy, vz
+
+
+def _calc_beta_c(v, dg, ba):
+    """
+    Calculate the cross-axis tilt angle.
+
+    Parameters
+    ----------
+    v : tuple
+        tracker normal
+    dg : float
+        delta gamma, difference between axis and ground azimuths [degrees]
+    ba : float
+        axis tilt [degrees]
+
+    Returns
+    -------
+    beta_c : float
+        cross-axis tilt angle [radians]
+    """
+    vnorm = np.sqrt(np.dot(v, v))
+    beta_c = np.arcsin(
+        ((v[0]*cosd(dg) - v[1]*sind(dg)) * sind(ba) + v[2]*cosd(ba)) / vnorm)
+    return beta_c
+
+
+def calc_cross_axis_tilt(
+        slope_azimuth, slope_tilt, axis_azimuth, axis_tilt):
+    """
+    Calculate the angle, relative to horizontal, of the line formed by the
+    intersection between the slope containing the tracker axes and a plane
+    perpendicular to the tracker axes.
+
+    Use the cross-axis tilt to avoid row-to-row shade when backtracking on a
+    slope not parallel with the axis azimuth. The slope is assumed to be a
+    plane containing all tracker axes. Cross-axis tilt follows the same
+    right-handed convention as tracker rotation. For example a tracker with
+    axis azimuth facing south will have a negative cross-axis tilt if sloped
+    down to the east and positive if sloped up.
+
+    Parameters
+    ----------
+    slope_azimuth : float
+        direction of normal to slope on horizontal [degrees]
+    slope_tilt : float
+        tilt of normal to slope relative to vertical [degrees]
```

Comment:
```suggestion
    slope_azimuth : float
        direction of normal to tracker axis plane when projected to horizontal [degrees]
    slope_tilt : float
        tilt of normal to slope relative to vertical [degrees]
```
Here, `slope_tilt` is relative to vertical, akin to zenith. The output `cross_axis_tilt` is relative to horizontal. I think it would be better to have `slope_tilt` relative to horizontal to somewhat reduce the mental gymnastics to visualize this calculation.

Reply:
Can you please clarify? The `slope_azimuth` and `slope_tilt` specify the orientation of the slope not the trackers (unless you mean the plane defined by all the trackers == "slope"). It's hard for me to parse this suggested change, but I believe it introduces the term "tracker axis plane" now to represent what I thought we had decided to call the "slope" -- to be consistent with the NREL tech report (and which I was previously calling the "system plane"). IMHO if "system" is easily misinterpreted as referring to a single tracker (altho TBH how can a single line create a plane?) then I think "tracker axis plane" is equally likely to confused. Perhaps "tracker axes plane" plural? Might be too subtle?

>`slope_tilt` is relative to vertical

That's why I initially called this the `system_zenith` - IMO a plane's orientation is defined by its normal's zenith and azimuth. But I believe the normal's zenith is the same as the plane's tilt. It's just hard in my mind to define the azimuth of a plane without it's normal

Maybe we should change `slope_tilt` to `slope_zenith`?

Reply:
I'm sorry for the swirling terms, I do think we are spiraling inwards to agreement. "tracker axes plane" == "slope" =="system plane"" at least the idea is clear. Agree that "tracker axis plane" is not quite there.

`zenith` would be better than `tilt`, since it's an angle from vertical of the normal.